### PR TITLE
consistent page transitions

### PR DESCRIPTION
**Pull request by 7opf**
- Closes https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/issues/14
- Current page will fly off-screen left when navigating to the next page and flies off-screen right when navigating back in history revealing the target page underneath
- Fixes z-index issues

### CODE REVIEW

File: identity-enabler/deviceId-mobile-app/src/lib/ui/helpers.ts [hunk 1]
```diff
@@ -53,3 +55,18 @@ export function multiClick(node: HTMLElement, opts?: { numClicks?: number; maxDe
...
 }
+
+/**
+ * Wraps fly transition to ensure element is on top during outro or intro.
+ *
+ * @param node
+ * @param opts
+ */
+export function flyOver(node: HTMLElement, opts: FlyParams) {
+    const z = +getComputedStyle(node).zIndex;
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/src/lib/ui/helpers.ts:66]: performance-wise getComputedStyle is very costly

**Review by jmcanterafonseca-iota — CHANGES_REQUESTED** [review]
some reconsiderations needed

File: identity-enabler/deviceId-mobile-app/src/lib/ui/helpers.ts (hunk 1 above)
Comment (7opf) [reply identity-enabler/deviceId-mobile-app/src/lib/ui/helpers.ts:66]: In this case it's only called once prior to animation, I think it's safe usage.

**Review by jmcanterafonseca-iota — COMMENTED** [review]
I am still not convinced, relying on getComputedStyle is a good practice.

We put onhold and will try to resume it later if we find time

**Comment by 7opf** [comment]
> I am still not convinced, relying on getComputedStyle is a good practice.
> 
> We put onhold and will try to resume it later if we find time

@jmcanterafonseca-iota Latest commit removes usage of getComputedStyle

**Pull request closed without merging**